app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,6 +1,11 @@
 class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
   include IdmeAuthenticatable
 
+  def idme_sign_in(after_login: nil)
```

Comment:
Won't `after_login` be in the `params` here (instead of being passed in as an argument to the controller method?)